phpstan-baseline.neon

```diff
@@ -3386,54 +3386,14 @@ parameters:
 			path: modules/custom/social_gdpr/social_gdpr.module
 
 		-
-			message: "#^Function social_graphql_install\\(\\) has no return type specified\\.$#"
-			count: 1
-			path: modules/custom/social_graphql/social_graphql.install
-
-		-
-			message: "#^Function social_graphql_update_10101\\(\\) has no return type specified\\.$#"
-			count: 1
-			path: modules/custom/social_graphql/social_graphql.install
-
-		-
-			message: "#^Function social_graphql_update_dependencies\\(\\) has no return type specified\\.$#"
-			count: 1
-			path: modules/custom/social_graphql/social_graphql.install
-
-		-
-			message: "#^Call to an undefined method Drupal\\\\Core\\\\Entity\\\\Query\\\\QueryInterface\\:\\:sortAggregate\\(\\)\\.$#"
-			count: 1
-			path: modules/custom/social_graphql/src/GraphQL/EntityConnection.php
-
-		-
-			message: "#^Method Drupal\\\\social_graphql\\\\GraphQL\\\\EntityConnection\\:\\:getResult\\(\\) should return GraphQL\\\\Executor\\\\Promise\\\\Adapter\\\\SyncPromise but returns GraphQL\\\\Executor\\\\Promise\\\\Adapter\\\\SyncPromise\\|null\\.$#"
-			count: 1
-			path: modules/custom/social_graphql/src/GraphQL/EntityConnection.php
-
-		-
-			message: "#^Parameter \\#1 \\$result of method Drupal\\\\social_graphql\\\\GraphQL\\\\ConnectionQueryHelperInterface\\:\\:getLoaderPromise\\(\\) expects array, array\\|int given\\.$#"
-			count: 1
-			path: modules/custom/social_graphql/src/GraphQL/EntityConnection.php
-
-		-
-			message: "#^Method Drupal\\\\social_graphql\\\\Plugin\\\\GraphQL\\\\DataProducer\\\\Payload\\\\PayloadViolations\\:\\:resolve\\(\\) should return array\\<Drupal\\\\social_graphql\\\\GraphQL\\\\ViolationInterface\\>\\|null but returns array\\|null\\.$#"
+			message: "#^Method Drupal\\\\social_graphql\\\\Plugin\\\\GraphQL\\\\DataProducer\\\\Payload\\\\PayloadViolations\\:\\:resolve\\(\\) should return Drupal\\\\social_graphql\\\\GraphQL\\\\ViolationInterface\\|null but returns array\\|null\\.$#"
```

Comment:
This ignored error is actually a bug now that you made it explicit :D (See also my comment on that function ;-) )